frontend/src/reducers/form.js

```diff
@@ -42,29 +41,34 @@ const profileInitialState = {
 
 function profile(state=profileInitialState, action={}) {
   switch (action.type) {
-    case constants.SET_EDIT_MODE_PROFILE:
-      if (!action.isEditMode) {
-        return merge({}, profileInitialState, { isEditMode: action.isEditMode });
-      }
-      return merge({}, state, { isEditMode: action.isEditMode });
+    // TODO constants.SET_EDIT_MODE_PROFILE doesn't exist, should it?
```

Comment:
you can remove all of the constants from this reducer that don't exist. `APPEND_PROFILE_FORM` is the only one we are using.